app/src/main/res/values/strings.xml

```diff
@@ -2,16 +2,16 @@
 <resources>
 
     <!-- Applications list -->
-    <string name="info_applications_list_refreshed">Applications list refreshed</string>
-    <string name="button_refresh_list">Refresh the applications list</string>
-    <string name="error_application_not_found">The application %1$s has not been found!</string>
+    <string name="info_applications_list_refreshed">App list refreshed</string>
+    <string name="button_refresh_list">Refresh app list</string>
+    <string name="error_application_not_found">Could not find the %1$s app.</string>
 
     <!-- Favorites -->
-    <string name="info_no_favorites_yet">You can add your favorites applications here\nby clicking the menu button.</string>
-    <string name="button_manage_favorites">Manage favorites</string>
+    <string name="info_no_favorites_yet">Add your favorite apps here\nby clicking the menu button.</string>
+    <string name="button_manage_favorites">Favorites</string>
     <string name="button_select_favorites">Select favorites</string>
-    <string name="hint_favorites_order">You can modify the favorites order in the list below (use the empty areas on sides to scroll).</string>

```

Comment:
Unmarked noun error

Reply:
Correction OK for me on words.
However, I found it more elegant to have a single phrase like this:
`Modify the order of favorites in the list below (use the sides to scroll).`

Reply:
I think that is sentence splicing. Going with "using" to sort of avoid that.

Reply:
Not sure what a sentence splicing is (you are definitely better in English grammar than me!) but the following is ok for me and indeed looks better:
`Modify the order of favorites in the list below using the sides to scroll.`

Reply:
Oh, I just saw below that it was already modified to `Modify the order of favorites in the list below (using the sides to scroll.)` (sorry, this is the first time I use pull request comments)

The dot inside the closing parenthesis looks like a typo to me, I would have rather written:
`Modify the order of favorites in the list below (using the sides to scroll).`

Reply:
Ah yes, it was two sentences (and the dot should be inside), but then I changed it. Good catch.